src/plugins/vis_builder/public/application/components/data_tab/field_bucket.test.tsx

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React from 'react';
+// @ts-ignore
+import { mountWithIntl } from 'test_utils/enzyme_helpers';
+import { IndexPatternField } from '../../../../../data/common';
+// @ts-ignore
+import { findTestSubject } from '@elastic/eui/lib/test';
```

Comment:
Curious are these ignored elsewhere?

Reply:
at least in that folder they are - [field_details.test.tsx](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/src/plugins/vis_builder/public/application/components/data_tab/field_details.test.tsx) and [/utils/get_field_details.test.ts](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/src/plugins/vis_builder/public/application/components/data_tab/utils/get_field_details.test.ts)

Reply:
since the field details component includes my field bucket component, I mostly used the setup found in [field_details.test.tsx](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/src/plugins/vis_builder/public/application/components/data_tab/field_details.test.tsx)